src/main/java/ch/wisv/areafiftylan/controller/AuthenticationController.java

```diff
@@ -61,6 +72,19 @@ public ModelAndView getLoginPage() {
         return createResponseEntity(HttpStatus.OK, "Here's your token!");
     }
 
+    @RequestMapping(value = "/token", method = RequestMethod.POST)
+    public ResponseEntity<?> createAuthenticationToken(@RequestBody UserDTO userDTO) {
+        String authToken = authenticationService.createNewAuthToken(userDTO.getUsername(), userDTO.getPassword());
+
+        return createResponseEntity(HttpStatus.OK, "Token successfully created", authToken);
+    }
+
+    @PreAuthorize("isAuthenticated()")
+    @RequestMapping(value = "/token/verify", method = RequestMethod.POST)
+    public ResponseEntity<?> verifyToken() {
```

Comment:
This method has been created under the assumption that if someone reaches this endpoint the token is valid by default? Maybe it's better to send the token as well, and actually perform a verification on the actual token to be fully sure?